### DISCUSSION TITLE
styling svg with border radius does not work

### DISCUSSION THREAD

**Discussion by gsgou** [wieslawsoltes/Svg.Skia#133] [General]
Same issue with the rendering of SVGs here:
https://github.com/mono/SkiaSharp/issues/2220

this renders correct
![de](https://user-images.githubusercontent.com/911534/185475451-21cab3bf-72da-492d-b486-802f012a99d9.svg)

while this does not
![de-de](https://user-images.githubusercontent.com/911534/185475380-01ea2662-c392-41d0-97ca-9acfb2baa2b8.svg)

See attached project (change resource from 1 to 2 accordingly)
[svg.zip](https://github.com/wieslawsoltes/Svg.Skia/files/9376851/svg.zip)

**Comment by mattleibow** [comment]
ngl, I can't see the difference... Am I blind?

**Comment by gsgou** [comment]
![image](https://user-images.githubusercontent.com/911534/185717100-a0d5fef9-b41b-42a9-bed2-b92681ec2004.png)

tks for checing on this @mattleibow 
by using embedded resource 1 you get the image rendered on the left
by using embedded resource 2 you get the image rendered on the right
both should look circular as rendered in this thread by Github.
Seems border radius or css styles maybe aren't interpreted correctly.

**Comment by mattleibow** [comment]
Maybe update the image in the issue description as I see 2 circles.

**Comment by gsgou** [comment]
The images in the issue description are the same as in the project uploaded.
While the 1st achieves the circular path with border-radius
the 2nd does it with a mask. The browser renders both correctly circular,
so should also Skia.Svg. But i can remove them if its misleading?

**Comment by mattleibow** [comment]
Ah I see. Maybe a table with each and then how skia renders it. That was we can see which is the issue.
